compass/ocean/tests/isomip_plus/geom.py

```diff
@@ -17,7 +17,7 @@ def define_thin_film_mask_step1(ds_mesh, ds_geom):
 
     ds_geom : xarray.Dataset
         Ice-sheet topography produced by
-        :py:func:`compass.ocean.tests.isomip_plus.geom.process_input_geometry()`
+        :py:class:`compass.ocean.tests.isomip_plus.process_geom.ProcessGeom`
```

Comment:
@cbegeman, above here, I think the docstirng just got copied from `interpolate_ocean_mask()`.  Maybe that's fine but can you double check?

Reply:
This change looks right. I think the function change happened with your reorganization cea496c4fda1a215fa27e0bb238e425b78c1c461.

Reply:
Yeah, there isn't a way to comment on lines that weren't changed.  This is very useful!